### ISSUE TITLE
New Contact us page for 508 review

### ISSUE THREAD

**Issue by bethpottsVADEPO** [department-of-veterans-affairs/va.gov-team#17542]
## Issue Description

Per a convo with Kevin H. and Jen L., our new Contact us page needs an accessibility review. This page is not linked to a new tool--it's a static content page. It will live at the root level and once it's published, we'll link to it from the header (in place of the drop down that's there now) and footer. The url will be: va.gov/contact-us

Here's the header/footer mock-up:
https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1603977230451/f2be5be57cb4c9ef848586f1ffa821890e3f1544 

Here's the page itself (you'll need to be on the network to view it): 
http://preview-prod.vfs.va.gov/preview?nodeId=8472 

---
## Tasks
- [ ] Determine if this page is 508 compliant.

## Acceptance Criteria
- [ ] Let public websites (Beth) know if there are any red flags re: accessibility

---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by 1Copenut** [comment]
Thank you @bethpotts . I'll pick this up for review on the 28th of December. I'll be able to do a fast first pass in the preview environment, and will finish with additional screen readers in production after you launch.

**Comment by bethpottsVADEPO** [comment]
Hi @1Copenut ! Do you have any questions for me about the header/footer entry points? There are a couple iterations contained in that mural mockup. The last screenshot you see at the bottom of the mural is the most up to date. There are only minor editorial changes to what's there, like removing the periods from the subheads like "Call us" and changing "Visit a medical center or benefits office" to "Visit a medical center or regional office." Hoping to publish the contact us page this week.

**Comment by 1Copenut** [comment]
@bethpotts Hi there. I haven't had a chance to review the new page yet. I was out all last week on vacation and had to be out of the office unexpectedly this afternoon. I'll try to give it a look late tonight.

**Comment by bethpottsVADEPO** [comment]
@1Copenut I'm so sorry, please don't lose sleep over this. I think it'll be straightforward, and whenever you can get to it will be great.

**Comment by 1Copenut** [comment]
@bethpotts I took a look tonight, and tested what could be done in the preview environment. I have 3-4 lower priority tickets to enter tomorrow, but logged the one item I'd like to see fixed before launch if possible, or as soon as possible post launch. LMK tomorrow if you have any questions or concerns.

**Comment by bethpottsVADEPO** [comment]
Makes sense. Thank you for the feedback. Is this what I should update it to?:
<a href="tel:711" aria-label=7 1 1.">711</a>

I see some instances on our site where the aria-label number has a period before the end quotation marks and some where there's no period. Which way is correct?

Also, I notice on our hub pages in the right rail that the 711 number is plain text. I'm assuming we should add the aria label to those? For example, see the "Ask questions" box under the COVID-19 promo box on the [health care hub](/health-care). We have this in plain text: If you have hearing loss, call TTY: 711.

**Comment by 1Copenut** [comment]
@bethpotts I commented on the issue ticket for the 711 link here https://github.com/department-of-veterans-affairs/va.gov-team/issues/17966#issuecomment-752996949 for visibility.

That's a great question about the TTY: 711 not being linked in the hub page right rail. I think it's worth asking the design team, maybe at next week's VA design meetup. I'll put an ask in on the design Slack channel.

**Comment by bethpottsVADEPO** [comment]
@1Copenut I just spoke with Danielle about this. She said the decision was made back in June to link 771, and it's probably just an oversight that it didn't happen in the right rail on all the hub landing pages. I'll alert the content team and they can get it done.

**Comment by bethpottsVADEPO** [comment]
@1Copenut I'll be on the look-out for the lower priority tickets you mentioned. The "must" fix is in, so we'll proceed with publishing this page now. Will fix any other issues you find when we get your tickets. Thank you!!

**Comment by 1Copenut** [comment]
@bethpotts I entered two more tickets, one for the table HTML, and one for the breadcrumb. I have done as much testing as I can on the VA network. I will do a more exhaustive screen reader test in production when you're ready. Best case scenario will be to test after #17984 is updated so I can get a clear picture of the table behavior with screen readers.

**Comment by bethpottsVADEPO** [comment]
Thank you, saw those tickets come through.

**Comment by 1Copenut** [comment]
@bethpotts Do you mind if we close this ticket and issue a full 508 audit request now that it's launched into production? I'll finish the screen reader testing there, and carry the one open item forward into that new epic.

Thank you!

**Comment by bethpottsVADEPO** [comment]
@1Copenut Sounds good, thank you for your attention to this!

**Comment by 1Copenut** [comment]
@bethpotts Is there a different ticket number I can be assigned to for the final review in production? Thank you!

**Comment by shiragoodman** [comment]
Hey @bethpottsVADEPO  - I've asked @1Copenut to keep this ticket open until we see the Full Accessibility & 508 Office Audit ticket come through. Thanks!

**Comment by bethpottsVADEPO** [comment]
@shiragoodman Sounds good. @RLHecht  is going to submit the full 508 audit request. Thanks!
